b2sdk/encryption/setting.py

```diff
@@ -204,8 +230,8 @@ def from_file_version_dict(cls, file_version_dict: dict) -> EncryptionSetting:
             return EncryptionSetting(EncryptionMode.NONE)
         key_id = None
         file_info = file_version_dict.get('fileInfo')
-        if file_info is not None:
-            key_id = file_info.get(SSE_C_KEY_ID)
+        if file_info is not None and SSE_C_KEY_ID_FILE_INFO_KEY_NAME in file_info:
+            key_id = urllib.parse.unquote(file_info[SSE_C_KEY_ID_FILE_INFO_KEY_NAME])
```

Comment:
again, two retrievals where one would be enough. Also, can file_info really be None here?